class/defaults.yml

```diff
@@ -951,7 +1023,47 @@ parameters:
                 cpu: "2"
                 memory: "8Gi"
                 disk: 16Gi
-
+        codey:
```

Comment:
I don't like the tfact that we are considering Codey as a service like any other service we already offer. Codey is a specific implementation of Forgejo so we need to discuss whether we want to differentiate it in the component from the normal services. For instance we should consider to put it under `forgeo` or maybe under `abstracService` instead of `service`. Also if we agreed that Codey has nothing to do with VSHN that it might not make sense to leave it under vshn at all (appcat.services.vshn). THis has to be discussed asap before it's too late.

Reply:
this is a good point and I fully agree, but we have our workflow in component and it expects services to be structured in a specific way, if we want to do any change, it's another story, another ticket and it requires a team-wide decision.